test-application/src/main/java/com/google/cloud/runtimes/stackdriver/StackDriverMonitoringService.java

```diff
@@ -0,0 +1,66 @@
+package com.google.cloud.runtimes.stackdriver;
+
+import com.google.api.Metric;
+import com.google.api.MetricDescriptor;
+import com.google.api.MonitoredResource;
+import com.google.cloud.monitoring.v3.MetricServiceClient;
+import com.google.cloud.monitoring.v3.PagedResponseWrappers;
+import com.google.monitoring.v3.*;
+import com.google.protobuf.util.Timestamps;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+
+@Service
+public class StackDriverMonitoringService {
+
+    private static Logger LOG = Logger.getLogger(StackDriverMonitoringService.class.getName());
+
+    public void createMetricAndInsertTestToken(String projectId, String metricType, long metricValue) throws IOException {
+        MetricServiceClient metricServiceClient = MetricServiceClient.create();
+        int retries = 3;
```

Comment:
I know it's a test app, but maybe make this constant a property in `application.properties`?